source/game_sa/CWorld.cpp

```diff
@@ -246,7 +248,39 @@ bool CWorld::SprayPaintWorld(CVector& posn, CVector& outDir, float radius, bool
 
 // Converted from cdecl void CWorld::RemoveFallenPeds(void) 0x565CB0
 void CWorld::RemoveFallenPeds() {
+#ifdef USE_DEFAULT_FUNCTIONS
     plugin::Call<0x565CB0>();
+#else
+    for (int i = CPools::ms_pPedPool->GetSize(); i; i--)
+    {
+        CPed* pPed = CPools::ms_pPedPool->GetAt(i - 1);
+        if (!pPed)
+            continue;
+
+        const CVector& vecPedPos = pPed->GetPosition();
+
+        if (vecPedPos.z > -100.0f)
+            continue;
+
+        if (!pPed->IsCreatedBy(ePedCreatedBy::PED_GAME) || pPed->IsPlayer())
+        {
+            // Find the closest path node to the ped
+            CNodeAddress pathNodeAddress;
+            ThePaths.FindNodeClosestToCoors(&pathNodeAddress, vecPedPos.x, vecPedPos.y, vecPedPos.z, 1, 1000000.0f, 0, 0, 0, 0, 0);
+            if (pathNodeAddress.m_wAreaId != -1)
+            {
+                const auto pathNodePos = ThePaths.GetPathNode(pathNodeAddress)->GetNodeCoors();
+                pPed->Teleport(pathNodePos, false); 
```

Comment:
```suggestion
                CVector pathNodePos = ThePaths.GetPathNode(pathNodeAddress)->GetNodeCoors();
                pathNodePos *= 0.125f;
                pathNodePos.z += 2.0f;
                pPed->Teleport(pathNodePos, false); 
```
You missed some code. Other than that, the PR is good.
![image](https://user-images.githubusercontent.com/10183157/99182210-6aa78900-2755-11eb-9620-14373d754f18.png)

Reply:
I didnt, because a path node's position is stored as a CompressedVector, and `getNodeCoors` returns the uncompressed `CVector`, hence why I left out the `* 0,125` part.
As for the `z + 2` I did forget about it, thanks!